phantazm-neuron-minestom/src/main/java/com/github/phantazmnetwork/neuron/bindings/minestom/entity/GroundMinestomDescriptor.java

```diff
@@ -0,0 +1,111 @@
+package com.github.phantazmnetwork.neuron.bindings.minestom.entity;
+
+import com.github.phantazmnetwork.api.VecUtils;
+import com.github.phantazmnetwork.commons.vector.Vec3I;
+import com.github.phantazmnetwork.neuron.agent.Agent;
+import com.github.phantazmnetwork.neuron.agent.GroundDescriptor;
+import com.github.phantazmnetwork.neuron.engine.PathContext;
+import com.github.phantazmnetwork.neuron.navigator.Controller;
+import com.github.phantazmnetwork.neuron.navigator.GroundNavigator;
+import com.github.phantazmnetwork.neuron.navigator.NavigationTracker;
+import com.github.phantazmnetwork.neuron.navigator.Navigator;
+import com.github.phantazmnetwork.neuron.node.GroundTranslator;
+import com.github.phantazmnetwork.neuron.node.NodeTranslator;
+import net.minestom.server.collision.CollisionUtils;
+import net.minestom.server.collision.PhysicsResult;
+import net.minestom.server.coordinate.Vec;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.instance.Instance;
+import net.minestom.server.instance.InstanceManager;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * A {@link MinestomDescriptor} for gravity-bound entities.
+ * @see GroundDescriptor
+ * @see MinestomDescriptor
+ */
+public interface GroundMinestomDescriptor extends MinestomDescriptor, GroundDescriptor {
+    @Override
+    default float getJumpHeight() {
+        return 1F;
+    }
+
+    @Override
+    default float getFallTolerance() {
+        return 3F;
+    }
+
+    /**
+     * Returns the maximum "step height" for this agent. For vertical distances greater less than the step height,
+     * the agent should teleport up instantaneously instead of jumping.
+     * @return the step height for this agent
+     */
+    default float getStepHeight() {
+        return 0.5F;
+    }
+
+    @Override
+    default @NotNull Vec3I computeTargetPosition(@NotNull Entity targetEntity) {
+        PhysicsResult result = CollisionUtils.handlePhysics(targetEntity, new Vec(0, -16, 0));
```

Comment:
why -16

Reply:
arbitrary value, but it should be reasonably small (larger distances = more blocks iterated and slower collision checks) but large enough so that players a few blocks off the ground (say, from jumping or falling some distance) don't cause path failures

Reply:
feel like that should get its own method?

Reply:
done